### ISSUE TITLE
Add more house info the map

### ISSUE THREAD

**Issue by RatingGull27** [RatingGull27/RatingGull27#4]
Add more house info the map

**Comment by werewolf26** [comment]
No, it does not work if I put more in the game see if a lag but player only allow joining in 0/18 player join the world but it is feature requests so yap
